source/transfer_list.rst

```diff
@@ -343,6 +343,23 @@ Inputs:
 
    #. Add `align8(te.hdr_size + te.data_size)` to `te_base_addr`.
 
+
+Adding a void TE
+^^^^^^^^^^^^^^^^
+
+Inputs:
+
+- `te_base_addr`: Base address where void TE to be added
+- `size`: Total size available for void TE
+
+#. Set `te.tag_id` (`te_base_addr + 0x0`) to `0x0` (XFERLIST_VOID)
+
+#. Set `te.hdr_size` (`te_base_addr + 0x3`) to `0x8`
+
+#. Set `te.data_size` (`te_base_addr + 0x4`) to `align8(size) - 0x8`
+
```

Comment:
If you want to mention optional data erasure in the "Remove" operation, you should probably do the same here?  (Note that this will change the area over which the checksum needs to be calculated. In fact, it may be easier to rewrite the "Remove" operation by referencing this one?)